Library/ENSideMenu.swift

```diff
@@ -192,35 +198,46 @@ open class ENSideMenu : NSObject, UIGestureRecognizerDelegate {
             sideMenuContainerView.addGestureRecognizer(rightSwipeGestureRecognizer)
             sourceView.addGestureRecognizer(leftSwipeGestureRecognizer)
         }
-
+        
+        // EXTEND 2/3: Hide menu option
+        if clickOutsideClose == true {
+            outerView.frame = CGRect(x: sideMenuContainerView.frame.width, y: 0, width: sourceView.frame.width - sideMenuContainerView.frame.width, height: sourceView.frame.height)
```

Comment:
`sideMenuContainerView`'s width can be updated dynamically (see `menuWidth` setter). in this case frame of `outerView`'s frame will be incorrect.